src/components/table/table.tsx

```diff
@@ -532,4 +532,18 @@ export class Table {
             </div>
         );
     }
+
+    private renderEmptyMessage() {
+        const showEmptyMessage =
```

Comment:
When we render the spinner we toggle the style `display` instead of not rendering it. That is why I choose to do the same here. 

The comment above the rendering of the spinner: 
```
{/* Toggle style instead of removing the loader
    because removing the element will cause a rerender, breaking the
    tabulator table */}
```